pages/galleri/styles.module.css

```diff
@@ -0,0 +1,14 @@
+.gallery {
+  width: 230px;
+  margin: 8px 20px 0;
+  display: -ms-grid;
```

Comment:
Do you need this?

Same with the other -ms properties below

Reply:
This file is unnecessary.